packages/nimbus/src/components/toggle-button-group/toggle-button-group.mdx

```diff
@@ -12,223 +12,442 @@ menu:
+      Compositions
+    </ToggleButtonGroup.Button>
+    <ToggleButtonGroup.Button id="filter-granted">
+      Granted
+    </ToggleButtonGroup.Button>
+    <ToggleButtonGroup.Button id="filter-veto">
+      Veto
+    </ToggleButtonGroup.Button>
+  </ToggleButtonGroup.Root>
 );
 ```
 
-## Controlled Mode
+### Offering many options
+
+In toolbars, button groups can organize related tools or actions.
 
-You can control the selection state externally:
+> [!TIP]\
+> **Do**
+>
+> -   Use expected tooling and order that a user will be familiar with when associated with effected content, like a rich-text editor.
+> -   Use consistent styling when creating toolbars or curated sets of tools for the user. Include tooltips to label icons.
+> -   The last button can be paired with a popover to feature additional options. Use the vertical three dot "More" icon for consistency throughout the UI.
 
 ```jsx-live
-const App = () => {
-  const [selectedKeys, setSelectedKeys] = React.useState(new Set(['controlled-center']));
-
-  return (
-    <Stack direction="column" gap="400">
-      <ToggleButtonGroup.Root
-        aria-label="Controlled example"
-        selectedKeys={selectedKeys}
-        onSelectionChange={setSelectedKeys}
-      >
-        <ToggleButtonGroup.Button id="controlled-left">Left</ToggleButtonGroup.Button>
-        <ToggleButtonGroup.Button id="controlled-center">Center</ToggleButtonGroup.Button>
-        <ToggleButtonGroup.Button id="controlled-right">Right</ToggleButtonGroup.Button>
-      </ToggleButtonGroup.Root>
-
-      <Text>Selected: {[...selectedKeys].join(', ') || 'none'}</Text>
-    </Stack>
-  );
-}
+const App = () => (
+  <ToggleButtonGroup.Root aria-label="Text formatting" selectionMode="multiple" selectedKeys={["format-italic"]} tone="primary">
+    <TooltipTrigger delay={0} closeDelay={0}>
+      <ToggleButtonGroup.Button id="format-bold" aria-label="Bold">
+        <Icons.FormatBold />
+      </ToggleButtonGroup.Button>
+      <Tooltip>Bold</Tooltip>
+    </TooltipTrigger>
+    <TooltipTrigger delay={0} closeDelay={0}>
+      <ToggleButtonGroup.Button id="format-italic" aria-label="Italic">
+        <Icons.FormatItalic />
+      </ToggleButtonGroup.Button>
+      <Tooltip>Italic</Tooltip>
+    </TooltipTrigger>
+    <TooltipTrigger delay={0} closeDelay={0}>
+      <ToggleButtonGroup.Button id="format-underline" aria-label="Underline">
+        <Icons.FormatUnderlined />
+      </ToggleButtonGroup.Button>
+      <Tooltip>Underline</Tooltip>
+    </TooltipTrigger>
+    <TooltipTrigger delay={0} closeDelay={0}>
+      <ToggleButtonGroup.Button id="format-strikethrough" aria-label="strikethrough">
+        <Icons.StrikethroughS />
+      </ToggleButtonGroup.Button>
+      <Tooltip>Strikethrough</Tooltip>
+    </TooltipTrigger>
+    <TooltipTrigger delay={0} closeDelay={0}>
+      <ToggleButtonGroup.Button id="options" aria-label="options">
+        <Icons.MoreVert />
+      </ToggleButtonGroup.Button>
```

Comment:
(continued from above)

This is what I'm referring to, it's simultaneously one of the toggle buttons and in the grouping, but it's also _not_ really a toggle button, more of a link to view additional options. I wonder if we should break it out of the group and make it a standalone icon button?